General queries/WD AV Signature and Platofrm Version.txt

```diff
@@ -0,0 +1,15 @@
+//Author: António Vasconcelos
+//Twitter: https://twitter.com/anthonws
+// This query provides you the latest signature and platform (MoCamp) for Windows Defender AV
+FileCreationEvents
+| where InitiatingProcessCommandLine contains "MpSigStub.exe"
```

Comment:
Performance: please make sure to filter by time:
`let startDate = ago(1d);`
and later on both queries on `FileCreationEvents` filter by it:
`and EventTime > startDate`